environment/prod/deployment/beta/.env.beta.secrets

```diff
@@ -8,6 +8,8 @@ DB_ROOT_PASSWORD=ENC[AES256_GCM,data:30TjrxKRNiGHo60eIbu48vsa048uQd6QhX2IK6Uw,iv
 MAIL_USERNAME=ENC[AES256_GCM,data:SWlgxtdEkdPrZUWa06EMuAbwMQ==,iv:A5IgnETj5k2Z3zE9KxiMwHLlRxreRhKPSel2LRUZmxY=,tag:VNjJoe0NW+jjFfxwU1LKsQ==,type:str]
 MAIL_PASSWORD=ENC[AES256_GCM,data:oq3a0D2U18c7EDSgX8WBwhVNizVS,iv:Z6dKNsvgY+H+cRuRuspziKK8eh0Fv1jwUOXT3K2A3GQ=,tag:ZWC1x8rokobND4+DlN4p5A==,type:str]
 ANALYTICS_URL=ENC[AES256_GCM,data:8pWAy5XJuc8KgeP8mGvKssqRS8CsKuMK1QtRT+uY1cNfUoe9Isl4tHENpW5aVls8vXHuJ6xzx/o9,iv:qS00PGf0cTBHtnYcXLwue/HZJtg4FAdDSl9Tb9XimYM=,tag:VZTj4S72Wd/5Bn/iaBVgug==,type:str]
+RECAPTCHAV3_SITEKEY=
+RECAPTCHAV3_SECRET=-
```

Comment:
Will you set it?

Reply:
I don't have access to this google project